tests/js/web/DataTables/dataTablesDrawCallbackTest.js

```diff
@@ -1,7 +1,7 @@
 const { drawCallback } = require('../../../../js/DataTables/dataTablesDrawCallback'),
     $ = require('jquery');
 
-describe('datatablesDrawCallback', () => {
+describe('dataTablesDrawCallback', () => {
     const infoStub = sinon.stub().returns({
         pages: 666
```

Comment:
😈

Reply:
Changed to match file name and to avoid confusion when grep'ing a single test